src/utils/updateValueWithTemplate.js

```diff
@@ -1,15 +1,20 @@
 import { isEmpty } from 'lodash';
 
 export const updateValueWithTemplate = (option, template) => {
-  const findPattern = /(?<=\*\*)(\w+)(?=\*\*)/g;
-  const replacePattern = /\*\*(\w+)\*\*/;
+  const findPattern = /\*\*(\w+)\*\*/g;
 
   let updatedString = template;
   const match = template.match(findPattern);
 
   if (!isEmpty(match)) {
     match.forEach(key => {
-      updatedString = updatedString.replace(replacePattern, option[key]);
+      const currentKey = key.replace(/\*\*/g, '');
+      const replacePattern = new RegExp(`\\**${currentKey}\\**`, 'g');
+      const templateHasOption = !!template.match(currentKey);
```

Comment:
```suggestion
      const templateHasOptions = !isEmpty(template.match(currentKey));
```

Reply:
What is the difference, except one more method is used ?

Reply:
As for me, it looks a little bit more readable, but it's up to you actually

Reply:
ok